types/apk-definition.json

```diff
@@ -1,37 +1,59 @@
 {
   "$schema": "https://packageurl.org/schemas/purl-type-definition.schema-1.0.json",
-  "$id": "https://packageurl.org/types/bitbucket-definition.json",
+  "$id": "https://packageurl.org/types/apk-definition.json",
   "type": "apk",
   "type_name": "APK-based packages",
   "description": "Alpine Linux APK-based packages",
   "repository": {
     "use_repository": true,
-    "note": "There is no default package repository; this should be implied either from the distro qualifiers key  or using a repository base url as repository_url qualifiers key."
+    "note": "There is no default package repository; this should be implied either from the distro qualifiers key or using a repository base url as repository_url qualifiers key."
   },
   "namespace_definition": {
     "requirement": "required",
     "note": "The namespace is the vendor such as alpine or openwrt. It is not case sensitive and must be lowercased.",
     "native_name": "vendor",
-    "case_sensitive": false
+    "case_sensitive": false,
+    "normalization_rules": [
+      "It is not case sensitive and must be lowercased."
+    ]
   },
   "name_definition": {
+    "requirement": "required",
     "note": "The name is the package name. It is not case sensitive and must be lowercased.",
     "native_name": "name",
-    "case_sensitive": false
+    "case_sensitive": false,
+    "normalization_rules": [
+      "It is not case sensitive and must be lowercased."
+    ]
   },
   "version_definition": {
-    "note": "The version is a package version as expected by apk.",
+    "requirement": "required",
```

Comment:
```suggestion
```

Installed packages have versions, but I don't understand why the PURL spec should be limited to talking about only specific versions of installed packages, especially if vers is going to be a related standard. How would you use vers if you can't name the package?